vendor/github.com/golang/snappy/AUTHORS

```diff
@@ -0,0 +1,15 @@
+# This is the official list of Snappy-Go authors for copyright purposes.
```

Comment:
I think all of this vendor directory can be added to git ignore - see `onos-topo` to a good example of how we maintain and build Go modules